lib/main_common.pm

```diff
@@ -388,6 +384,41 @@ sub load_svirt_vm_setup_tests {
     }
 }
 
+sub define_bootloader {
+    if (uses_qa_net_hardware() || get_var("PXEBOOT")) {
```

Comment:
you need to update version_utils import to state that you importing this function

Reply:
I think no, as in main_common.pm it is imported with 'use version_utils qw(:VERSION :BACKEND :SCENARIO);' which means, import everything:)

But in any case, this function should be removed. As Rodion guessed, it appeared here by mistake from other solution, that I decided to not use.

Reply:
ah this is what that magic string means :) thanks for clarification